interface/forms/eye_mag/js/eye_base.php

```diff
@@ -3946,12 +3946,11 @@ function show_by_setting() {
                   $('#left-panel').css("right","0px");
                   $('#EXAM_KB').css({position: 'fixed', top: '29px'});
                   $('#EXAM_KB').css('display', 'block');
-                  $('#EXAM_KB').draggable();
-                  $('#IMP').droppable({ drop: dragto_IMPPLAN } );
-                  $('#IMPPLAN_zone').droppable({ drop: dragto_IMPPLAN_zone } );
-                  $('#IMPPLAN_text').droppable({ drop: dragto_IMPPLAN_zone } );
```

Comment:
Hi @bradymiller 

I found some elements used the jQuery-UI droppable, Can you guide me how to trigger the element?

Reply:
hi @stu01509 ,

This is where it should work:
![image](https://user-images.githubusercontent.com/278968/82181924-49dea380-9898-11ea-9d18-f326796064d2.png)

So should be able to drag one of those diagnosis boxes to the left into the New Dx: text field. Note it does not work, however, in the main codebase :)
(all the other ways work such as double clicking the diagnosis)

Reply:
Thanks for your explanation, I will try it :)

Reply:
hi @stu01509 ,
Note that in above screenshot, should be able to do the following actions that will bring the diagnosis(es) from the right Impression/Plan Builder over to the left Impression/Plan section:
```
Drag a DX over by its handle: 
Double click on a DX`s handle: 
Multi-select desired DX(s) and click the icon
```
Currently none of those work. Rec trying it out on one of the development demos to see how it works. Let me know if you need a loom video to show.

Reply:
I know that currently didn't work, I am trying it on development demos and know that process, But it a little bit difficult to solve it, because it depends on a lot of jquery-ui function.

If we divide it into another issue, How do you think?

Reply:
hi @stu01509 , Definitely have option to migrate the droppable stuff into another issue/PR. If that is case, then would just remove all draggable/droppable stuff in this PR (note it does not work anyways in the main codebase) and then add back your stuff into another PR and continue to tinker with it there.

Reply:
although it sounds like from what you are saying that the other functionality (double click and multi-select) also rely on jquery-ui. If that is case, then we are stuck basically dealing with it here since removing jquery-ui . The positive is that you will turn into a js expert :)

Reply:
Ok, I got it,  I will try it on this PR :)